### ISSUE TITLE
Problem adding additional signing key using Builder

### ISSUE THREAD

**Issue by jimbobhickville** [StellarCN/py-stellar-base#93]
I think I've hit a bug in the Builder code. I have this code:

```
    add_signer_tx = stellar_builder.Builder(secret=issuer_seed, network=args.network)
    add_signer_tx.append_set_options_op(master_weight=3, low_threshold=1, med_threshold=3, high_threshold=3)
    add_signer_tx.append_set_options_op(signer_address=base_keypair.address().decode(), signer_weight=1)
    add_signer_tx.sign()
    add_signer_tx.submit()
```

But it throws the following exception when trying to create the operation for adding the signer_address:

```
Traceback (most recent call last):
  File "/home/jimbob/.virtualenvs/zipzap/lib/python3.5/site-packages/stellar_base/operation.py", line 306, in __init__
    decode_check('account', self.signer_address)
NameError: name 'decode_check' is not defined

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "./bin/issue_asset", line 58, in <module>
    add_signer_tx.append_set_options_op(signer_address=base_keypair.address(), signer_weight=1)
  File "/home/jimbob/.virtualenvs/zipzap/lib/python3.5/site-packages/stellar_base/builder.py", line 149, in append_set_options_op
    op = SetOptions(opts)
  File "/home/jimbob/.virtualenvs/zipzap/lib/python3.5/site-packages/stellar_base/operation.py", line 307, in __init__
    except DecodeError:
NameError: name 'DecodeError' is not defined
```

**Comment by jimbobhickville** [comment]
Appears to be because I didn't pass a `signer_type`, so I can work around it, but probably should fix this code as most people aren't going to pass a signer_type since that's not in the API documentation:

https://github.com/StellarCN/py-stellar-base/blob/v0.2/stellar_base/operation.py#L558-L564

**Comment by overcat** [comment]
Thanks for your advice, but the testnet seems to be down, I will fix it after it recovers.

**Comment by overcat** [comment]
fixed in v0.2, thank you.
